Content.Client/Administration/BwoinkSystem.cs

```diff
@@ -25,6 +26,7 @@ public sealed class BwoinkSystem : SharedBwoinkSystem
         [Dependency] private readonly IClientAdminManager _adminManager = default!;
         [Dependency] private readonly IPlayerManager _playerManager = default!;
         [Dependency] private readonly IClyde _clyde = default!;
+        [Dependency] private readonly IGameHud _hud = default!;
```

Comment:
Sorry smez but it's been 2 months and this tilts the shit outta me.

Reply:
:salute: